pkg/network/ebpf/c/protocols/postgres/decoding.h

```diff
@@ -211,8 +252,13 @@ int socket__postgres_process(struct __sk_buff* skb) {
         bpf_tail_call_compat(skb, &protocols_progs, PROG_POSTGRES_PROCESS_PARSE_MESSAGE);
         return 0;
     }
-
-    postgres_entrypoint(pkt, &conn_tuple, &header, NO_TAGS);
+    __s8 result = postgres_entrypoint(pkt, &conn_tuple, &header, NO_TAGS);
+    postgres_kernel_msg_count_t* pg_msg_counts = get_pg_msg_counts_map(pkt);
+    if (pg_msg_counts != NULL) {
```

Comment:
redundant check, you already do that inside `update_pg_message_count_telemetry`

Reply:
I've changed this code